README.md

```diff
@@ -71,6 +71,58 @@ env "adapter.config.file.path=/opt/xs2a-adapter/custom-config/custom.adapter.con
             --timeout-request 3000
     ```
 
+## Using XS2A Adapter as a library
```

Comment:
I propose to add more info about:

- how to run the adapter with a specific bank adapter
- highlight that aspsp-registry contains only info about sandboxes, if you need production configuration. Please contact with adorsys sales team. something like that